lib/routes/rides/routes.js

```diff
@@ -0,0 +1,118 @@
+import get from 'lodash/get';
+import { notFound, badImplementation, badRequest } from '@utils/responseInterceptors';
+import Joi from 'joi';
+import { bookARide, getPendingRides, assignDriverToRide } from '@daos/ridesDao';
+import { customerIdSchema } from '@utils/validationUtils';
+
+export default [
+  {
+    method: 'POST',
+    path: '/book/{customerId}',
```

Comment:
```suggestion
    path: '/',
```
Lets send the customerId in body itself